File: hello-spark2.3.0-java1.8-maven/src/main/java/com/hello/spark/java/mllib/HelloSparkMLlib.java
package com.hello.spark.java.mllib;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;

public class HelloSparkMLlib {
	public static void main(String[] args) {
		// 初始化
		SparkConf sparkConf = new SparkConf().setMaster("local[*]").setAppName("SparkJavaWordCountLocal");
		JavaSparkContext sparkContext = new JavaSparkContext(sparkConf);


	}

	/**
	 * 相关性分析
	 *
	 * @param sparkContext
	 */
	private static void correlationAnalysis(JavaSparkContext sparkContext){

	}
}

File: hello-spark2.3.1-scala2.11.12-maven/src/main/scala/com/hello/spark/scala/mllib/HelloSparkMLlib.scala
package com.hello.spark.scala.mllib

import org.apache.spark.{SparkConf, SparkContext}

object HelloSparkMLlib {
  def main(args: Array[String]): Unit = {
    // 初始化
    val conf = new SparkConf().setMaster("local").setAppName("HelloSparkMLlib-scala")
    val sc = new SparkContext(conf)

    val sparkMLlib = new HelloSparkMLlib

    sparkMLlib.correlationAnalysis(sc);
  }

}
class HelloSparkMLlib {
  /**
    * 相关性分析
    *
    * @param sparkContext
    */
  def correlationAnalysis(sparkContext: SparkContext): Unit = {
    val sc = sparkContext



  }
}